lib/package/config.coffee

```diff
@@ -510,6 +510,15 @@ config =
     type: 'array'
     default: ['source.julia', 'source.weave.md', 'source.weave.latex']
     order: 6
+  disableProxy:
```

Comment:
This seems better to be under `Julia Options`, because all the other configs around Julia process is so ?

Reply:
It's not strictly for Julia though (and has an impact on all internal terminals, not just the REPL).

Reply:
yep, just asked it for a case if this was only intended to fix the package-installation issue.